src/main/java/com/gpuopenanalytics/jenkins/remotedocker/job/DockerImageConfiguration.java

```diff
@@ -46,24 +46,45 @@
 public class DockerImageConfiguration extends AbstractDockerConfiguration {
 
     private final String image;
-    private final boolean forcePull;
+    private final String maxRetries;
+    private final boolean pullImage;
 
     @DataBoundConstructor
     public DockerImageConfiguration(List<ConfigItem> configItemList,
                                     List<VolumeConfiguration> volumes,
                                     String image,
-                                    boolean forcePull) {
+                                    ForcePull forcePull) {
```

Comment:
Doing it this way will break existing job configurations when the plugin is updated. The constructor should take two parameters `forcePull` and `maxRetries` and the jelly should bind to those.